hal/halx86/generic/sysinfo.c

```diff
@@ -12,8 +12,50 @@
 #define NDEBUG
 #include <debug.h>
 
+NTSTATUS
+NTAPI
+HaliHandlePCIConfigSpaceAccess(BOOLEAN IsRead,
+                               ULONG Port,
+                               ULONG Length,
+                               PVOID Buffer);
+
+AMLI_ILLEGAL_IO_PORT_ADDRESSES AMLIIllegalIOPortAddresses[19] =
```

Comment:
You don't need to specify the 19, the compiler will use the length of the initializer.

Reply:
Fixed.